test/kitchen/test/integration/system-probe-test/rspec_datadog/system-probe-test_spec.rb

```diff
@@ -13,13 +13,23 @@
   "pkg/collector/corechecks/ebpf/probe"
 )
 
+# most prebuild tests are not running on arm64
+# as tests run only on >=5.5.0 kernels and (runtime or CORE)
+arm64_tests = Array.[](
+  "pkg/network/protocols/events",
```

Comment:
Are these `arm64` only? Why not include them in `runtime_compiled_tests` and `co_re_tests`?

Reply:
We may do that automatically in the future

Reply:
Yeah, but you are only running them if `arm64`? Why not also for `amd64/x86_64`?

Reply:
Oh, I see you updated the list.